plugins/task-resources/src/components/kanban/KanbanView.svelte

```diff
@@ -207,7 +221,7 @@
             {/each}
           </KanbanPanel>
         {/each}
-        <!-- <KanbanPanelEmpty label={'Add new column'} /> -->
+        <KanbanPanelEmpty on:add={(e) => addItem(e.detail)}/>
```

Comment:
Better to use slot and allow to pass KanbanPanelEmpty if required for Kanban users, since this functionality will be optional.

Reply:
Done